script/create_admin.rb

```diff
@@ -40,34 +40,27 @@ def multi_gets(all_text = "")
 
 list = CSV.parse(multi_gets)
 
-puts "\nFor new admins, copy and paste each section into a separate file and upload to the admin's Vault:\n"
-
 admins = []
 list.each do |user|
   login = user[0].gsub(/\s+/, "")
   email = user[1]&.strip
   roles = user.drop(2).compact.map(&:strip).map(&:downcase)
 
   a = Admin.find_or_initialize_by(login: "admin-#{login}")
+  success_message = a.new_record? ? "Created and notified" : "Updated"
   a.email = email if email.present?
   a.roles = roles if roles.present?
 
+  # If this is a new admin, we need to set a temporary password.
   if a.new_record?
-    # Create password only for new admins
-    password = `pwgen 8 1`.strip
+    password = `pwgen 10 1`.strip
```

Comment:
As an aside and as out of scope this pains me :)

```
2.7.3 :007 > (0...10).map{ [*'0'..'9',*'A'..'Z',*'a'..'z'].sample}.join
 => "Cj1Q0JCp0I"
``` 
from https://caseyscarborough.com/blog/2013/07/23/generating-alphanumeric-strings-in-ruby/

Reply:
I'd use [`SecureRandom.alphanumeric`](https://ruby-doc.org/stdlib-2.7.3/libdoc/securerandom/rdoc/SecureRandom.html) and avoid implementing our own.

If we do this, let's remove `pwgen` as a dev dependency as well:

https://github.com/otwcode/otwarchive/blob/b63fce67a3975ec0614bb535c75d76c8dafbda71/config/docker/Dockerfile#L9

Reply:
Definitely game to do that! I saw someone use Devise's token generator to create the temporary, unshared keysmash password and I thought that was very clever, but SecureRandom is actually meant for such purposes, so we'll go with that.